src/draw/lv_draw_vector_private.h

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file lv_draw_vector_private.h
+ *
+ */
+
+#ifndef LV_DRAW_VECTOR_PRIVATE_H
+#define LV_DRAW_VECTOR_PRIVATE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_draw_vector.h"
+
+#if LV_USE_VECTOR_GRAPHIC
+
```

Comment:
All of the structures in this file need to have functions to set each and every single field otherwise they will not be usable to a binding.